### ISSUE TITLE
haskell-tool-cli-tests: Missing test suite files

### ISSUE THREAD

**Issue by lwm** [haskell-tools/haskell-tools#225]
Saw this failure on latest Stackage nightly:

```
haskell-tools-cli-tests: bench-tests/full-1.txt: openFile: does not exist (No such file or directory)
```

**Comment by nboldi** [comment]
The problem should be solved with the new haskell-tools-*:0.4.1.1 version.

**Comment by juhp** [comment]
Thanks, I think this can closed now.
